contents/blog/trademarks-blog-post.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-08-18
+title: Don’t Bother Securing Your Trademarks In The Beginning
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+Don’t bother securing your trademarks in the early days of your business. 
```

Comment:
At the beginning, let's add a jokey disclaimer like _This is intended as a super short and tactical guide to getting your trademarks sorted out for the first time, on a minimal budget. This is **not** legal advice and we are not lawyers - this is just what has worked for us!_

Reply:
Added